main.go

```diff
@@ -1,7 +1,13 @@
 package main
 
-func main()  {
-	i:=0
-	i+=1
+func Add() int {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`Add` is unused (deadcode)